.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+    unit-tests:
+        name: Unit tests
+        runs-on: ubuntu-latest
```

Comment:
Does it needs to be ubuntu? Isn't alpine faster / smaller?

Reply:
Sorry not possible I'm disappointed as well.

![image](https://user-images.githubusercontent.com/6873972/105080318-efce5780-5a90-11eb-84c1-03e50348acd2.png)

Reply:
And I cannot merge anything cause I don't have write access on this repo 😄

Reply:
Merged and I will change that.